.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,64 @@
+name: Bygg og deploy branch til test
+
+on:
+  push:
+    branches:
+    - deploy-dev/*
```

Comment:
Nå er det ser det ut til å være tre begreper for samme miljø - test, dev og preprod. Kan vi bruke preprod hele veien, så det blir konsistent? Ellers: Looks good to me, uten at jeg har vært borti dette før 😅